src/parser/core/modules/Gauge/CounterGauge.ts

```diff
@@ -37,11 +37,15 @@ export class CounterGauge extends AbstractGauge {
 
 	private history: CounterHistory[] = []
 
+	public get value(): number {
```

Comment:
```suggestion
	get value(): number {
```

(ts stuff is public by default)